core/common/src/main/java/alluxio/wire/MasterWebUIBrowse.java

```diff
@@ -26,7 +26,7 @@
  */
 @NotThreadSafe
 public final class MasterWebUIBrowse implements Serializable {
-  private static final long serialVersionUID = 6589358568781503724L;
+  private static final long serialVersionUID = 0L;
```

Comment:
This is because I had copied this number from another place, and then copy-pasted it throughout... They were not unique. I am not sure how these get generated.

Reply:
You can just pick a random long, or have the IDE select one, however for the ones that already exist, we should keep them the same.

Reply:
got it, this is done